File: src/poker/nlhe2max/HoldEmBotIO.java
public class HoldEmBotIO extends BotIO {

    public HoldEmBotIO(int botNumber) {
        super(botNumber);
    }

    public String sendToBot(String s) {
        if (botNumber == 1)
            return ((Bot1)bot).processMatchInfo(s);
        return ((Bot2)bot).processMatchInfo(s);
    } 
    
    public Object createBot() {
        if (botNumber == 1)
            return (new Bot1());
        return (new Bot2());
    }

}

File: src/poker/nlhe2max/HoldEmBotIO.py
from BotIO import BotIO

class HoldEmBotIO(BotIO):

    def __init__(self, botNum):
        BotIO.__init__(self, botNum) 

    def getBotResponse(self,s):
        if self.botNum == 1:
            return self.bot.processMatchInfo(s)
        return self.bot.processMatchInfo(s)

    def createBot(self):
        if self.botNum == 1:
            from Bot1 import Bot1
            return Bot1()
        from Bot2 import Bot2
        return Bot2()
